DEPRECATED.md

```diff
@@ -0,0 +1,33 @@
+C++ Log Server Deprecation Notice
+---------------------------------
+
+The CT log server implementation in this repository is no longer under
+active development. We recommend that new deployments use the new Go
+based server, which can handle much bigger trees:
+
+[CT Personality](https://github.com/google/certificate-transparency-go)
+[Trillian Log Backend](https://github.com/google/trillian)
+
+The C++ log server implementation in this repository keeps its Merkle tree in
+memory, which means two things: before it can handle requests, it has to load
```

Comment:
Maybe break the two things into bullets?

Reply:
Done.